widgetssdk/src/main/res/values/themes.xml

```diff
@@ -94,7 +94,7 @@
         <item name="gliaBaseDarkColor">@color/glia_base_dark_color</item>
         <item name="gliaBaseLightColor">@color/glia_base_light_color</item>
         <item name="gliaBaseNormalColor">@color/glia_base_normal_color</item>
-        <item name="gliaBaseShadeColor">@color/glia_base_shade_color</item>
+        <item name="gliaBaseShadeColor">@color/glia_base_light_color</item>
```

Comment:
Will this change affect the existing users?

Reply:
Fixed - https://github.com/salemove/android-sdk-widgets/pull/396/commits/40a661799a5ee54a4c0cf24786ef79e1216fb79d